Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM concourse/concourse-ci
+FROM ubuntu
 
 ENV SPRUCE_VERSION=1.5.0 \
   CF_CLI_VERSION=6.13.0 \
   VAULT_VERSION=0.6.0 \
   GENESIS_VERSION=1.5.2
 
 # base packages
-RUN apt-get install -yy curl file
+RUN apt-get update \
+      && apt-get install -yy curl file unzip git ruby \
```

Comment:
@lnguyen  will this get the right version of ruby for bosh_cli, or did we need to do other things to get this working for GE?